src/rippled/lib/rippled.js

```diff
@@ -303,6 +303,74 @@ const getNFTInfo = (rippledSocket, tokenId) =>
     return resp
   })
 
+const getBuyNFToffers = (rippledSocket, tokenId, limit = 50, marker = '') =>
+  queryP2P(rippledSocket, {
+    command: 'nft_buy_offers',
+    nft_id: tokenId,
+    limit,
+    marker: marker !== '' ? marker : undefined,
+  }).then((resp) => {
+    if (resp.error === 'objectNotFound') {
+      throw new Error('NFT not found', 404)
+    }
+    if (resp.error_message) {
+      throw new Error(resp.error_message, 500)
+    }
+    return resp
+  })
+
+const getSellNFToffers = (rippledSocket, tokenId, limit = 50, marker = '') =>
```

Comment:
These two should be shortcuts for one method that just takes the command since otherwise they are identical